testsuite/WIKI.md

```diff
@@ -0,0 +1,198 @@
+
+You can run all tests in serial from the python interpreter like this:
+```python
+from MDAnalysis.tests import test
+test()
+```
+
+Fore more details see below.
+
+## Examples ##
+Examples for output in various modes. For debugging, verbose mode is more useful as one can identify failing tests while they are running.
+
+### Serial testing ###
+For example, a successful test might look like the following
+```
+>>> MDAnalysis.tests.test()
+......S...S............................................................................................................................................................K.KK...........................................................................................................................................................................K...............................................................................................................................................................................................................K..............................................................K............................................................................................................................................................................................................................................................................................................................................................................K......K....................K.....................K...................K....................K...........................K...................K...................K.............................................................K...................K........................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................
+============ 5497 passed, 31 skipped, 1 xfailed in 1056.77 seconds =============
+```
+
+### Coverage ###
+We test code coverage of the unit tests with the  [coverage](http://nedbatchelder.com/code/modules/rees-coverage.html) plugin of nose. Currently, this is done automatically as part of the Travis job on Python 2.7 and is viewable on **[coveralls](https://coveralls.io/r/MDAnalysis/mdanalysis?branch=develop)**.
+
+If you want to generate a coverage report manually you can run
+```
+cd testsuite
+pytest --numprocesses 4 --cov=MDAnalysis --cov-report html
```

Comment:
`--numprocesses` needs `pytest-xdist`.

Either say how to install it or (my preferred option, but see discussion at https://groups.google.com/d/msg/mdnalysis-devel/uu0dFW5fYJ8/HCcbxjVNBQAJ ) it should be installed with `setup.py`.